django/core/management/commands/showmigrations.py

```diff
@@ -88,14 +91,19 @@ def show_list(self, connection, app_names=None):
             if not shown:
                 self.stdout.write(" (no migrations)", self.style.ERROR)
 
-    def show_plan(self, connection):
+    def show_plan(self, connection, app_names=None):
```

Comment:
The major difference between `migrate` and `showmigrations` is that the former only ever takes a single app name. What are your thoughts about having support for multiple app labels as you did right now? Is this by coincidence. Intentional. I would have thought of just mimicking the `migrate` API with a single app label as opposed to multiple.